protocol/consistencychecks_test.go

```diff
@@ -225,3 +238,118 @@ func TestVerifyKeyLookupResponseWithTB(t *testing.T) {
 		t.Error(e2)
 	}
 }
+
+func TestLookupBeforeMonitoring(t *testing.T) {
+	d, pk := NewTestDirectory(t, true)
+	cc := NewCC(d.LatestSTR(), pk, make(map[string]uint64), true)
+	registerAndVerify(d, cc, alice, key)
+
+	// monitor binding was inserted
+	d.Update()
+
+	// do a lookup before the clock tells us to do monitoring
```

Comment:
> `the binding returned to a lookup == the binding returned for monitoring`

This check is implicit in `lookupAndVerify` and `monitorAndVerify` with `alice, key` parameters. But this should be removed because of #173.